policyengine_us/parameters/gov/states/ar/tax/income/credits/cdcc/match.yaml

```diff
@@ -0,0 +1,12 @@
+description: Arkansas matches this percentage of the federal child care credit. 
+metadata:
+  unit: /1
+  period: year
+  label: Arkansas household and dependents care credit match
+  reference:
+    - title: Ark. Code R. § 26-51-502(c)(1)(A)(ii)
+      href: https://codes.findlaw.com/ar/title-26-taxation/ar-code-sect-26-51-502/
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions | line 35
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=15
```

Comment:
Cite a reference that specifies 20%. References must always support parameter values.